argoverse/visualization/visualize_sequences.py

```diff
@@ -14,6 +15,7 @@
 
 _ZORDER = {"AGENT": 15, "AV": 10, "OTHERS": 5}
 
+_PathLike = Union[str, "os.PathLike[str]", None]
```

Comment:
I think this would be more natural as:

```py
_PathLike = Union[str, "os.PathLike[str]"]
```

Then it is up to a user to use an `Optional[_PathLike]` if the `None` is permissable.

Reply:
@mgilson-argo Thank you for your review, and I modified the PathLike expression based on your advice.